ptero_workflow/api/v1/api.py

```diff
@@ -12,6 +12,14 @@
 api.add_resource(views.WorkflowDetailView,
     '/workflows/<int:workflow_id>', endpoint='workflow-detail')
 
+api.add_resource(views.ExecutionInputsView,
+    '/executions/<int:execution_id>/inputs',
+    endpoint='execution-inputs')
+
+api.add_resource(views.ExecutionOutputsView,
```

Comment:
I don't really understand why there are separate endpoints for execution inputs and outputs.

It seems more intuitive (and more RESTful) to simply have one resource for the execution and `PATCH` the outputs to set them.